### PR TITLE
Improved rubocop rules

### PR DESCRIPTION
**Pull request by KrauseFx**
Very much simplified the `rubocop.yml`, so that we can copy it over to newly generated plugins. 
This will also update the existing code base 👍

### CODE REVIEW

**Comment by KrauseFx** [comment]
Oh, @houndci-bot doesn't like the `.` rule, I'll revert that rule then

File: fastlane/lib/fastlane/actions/gradle.rb [hunk 1]
```diff
@@ -58,8 +58,8 @@ def self.run(params)
         apk_search_path = File.join(project_dir, '*', 'build', 'outputs', 'apk', '*.apk')
 
         # Our apk is now built, but there might actually be multiple ones that were built if a flavor was not specified in a multi-flavor project (e.g. `assembleRelease`), however we're not interested in unaligned apk's...
-        new_apks = Dir[apk_search_path].reject { |path| path =~ /^.*-unaligned.apk$/i}
-        new_apks = new_apks.map { |path| File.expand_path(path)}
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/gradle.rb]: ah! this feels a little weird that there is an opening space but not a trailing one in the closure.

File: fastlane/lib/fastlane/actions/gradle.rb [hunk 2]
```diff
@@ -58,8 +58,8 @@ def self.run(params)
...
+        new_apks = Dir[apk_search_path].reject { |path| path =~ /^.*-unaligned.apk$/i }
+        new_apks = new_apks.map { |path| File.expand_path(path) }
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/gradle.rb]: ah! this feels a little weird that there is an opening space but not a trailing one in the closure.

File: fastlane/lib/fastlane/actions/restore_file.rb [hunk 3]
```diff
@@ -5,7 +5,7 @@ def self.run(params)
         path = params[:path]
         backup_path = "#{path}.back"
         UI.user_error!("Could not find file '#{backup_path}'") unless File.exist? backup_path
-        FileUtils.cp(backup_path, path, {preserve: true})
+        FileUtils.cp(backup_path, path, { preserve: true })
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/restore_file.rb]: in contrast to the one above, this was corrected to having leading and trailing spaces. 👍

File: fastlane/lib/fastlane/actions/verify_build.rb [hunk 4]
```diff
@@ -85,7 +85,7 @@ def self.print_values(values)
             columns << key
             columns << case value
                        when Hash
-                         value.map {|k, v| "#{k}: #{v}"}.join("\n")
+                         value.map { |k, v| "#{k}: #{v}" }.join("\n")
```
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/verify_build.rb]: And this closure got a trailing one. It feels like something went a bit wrong with the first ones I mentioned above.

File: fastlane/lib/fastlane/plugins/plugin_info.rb [hunk 5]
```diff
@@ -39,9 +39,9 @@ def get_binding
 
     def ==(other)
       @plugin_name == other.plugin_name &&
-      @author == other.author &&
-      @email == other.email &&
-      @summary == other.summary
+        @author == other.author &&
+        @email == other.email &&
+        @summary == other.summary
```
Comment (ohwutup) [inline fastlane/lib/fastlane/plugins/plugin_info.rb]: 👍

File: fastlane/lib/fastlane/actions/gradle.rb (hunk 2 above)
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/gradle.rb]: There is a trailing and opening space. Or what do you mean? line 62?

Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/gradle.rb]: Nope, this looks all fine 👍

File: fastlane_core/lib/fastlane_core/update_checker/update_checker.rb [hunk 6]
```diff
@@ -135,8 +135,8 @@ def self.android_app_identifier(args, gem_name)
     def self.android_app_identifier_arg?(gem_name, arg)
       return arg == "--package_name" ||
              arg == "--app_package_name" ||
```
Comment (ohwutup) [inline fastlane_core/lib/fastlane_core/update_checker/update_checker.rb]: Seems like a space should have been added here maybe?

File: fastlane/lib/fastlane/actions/gradle.rb (hunk 2 above)
Comment (ohwutup) [inline fastlane/lib/fastlane/actions/gradle.rb]: yup! you're right! i misread it! my bad... ignore the rest of the comments that reference this one! sorry!

**Comment by mfurtak** [comment]
😮 Looks OK

**Comment by KrauseFx** [comment]
Closed with https://github.com/fastlane/fastlane/pull/5009

**Pull request closed without merging**